handlers/handlers.go

```diff
@@ -19,10 +19,11 @@ type Filter struct {
 	val                  Validator
 	downloadServiceURL   string
 	EnableDatasetPreview bool
+	EnableLoop11         bool
 }
 
 // NewFilter creates a new instance of Filter
-func NewFilter(r Renderer, fc FilterClient, dc DatasetClient, hc HierarchyClient, sc SearchClient, val Validator, downloadServiceURL string, enableDatasetPreview bool) *Filter {
+func NewFilter(r Renderer, fc FilterClient, dc DatasetClient, hc HierarchyClient, sc SearchClient, val Validator, downloadServiceURL string, enableDatasetPreview bool, enableLoop11 bool) *Filter {
```

Comment:
Consistency,
It doesn't matter too much whether cascading argument types are used or explicit types for each argument but ideally would be beneficial if the same was done within a file. In this instance cascading is being used so would be good to use that for the new enableLoop11, e.g.
```suggestion
func NewFilter(r Renderer, fc FilterClient, dc DatasetClient, hc HierarchyClient, sc SearchClient, val Validator, downloadServiceURL string, enableDatasetPreview, enableLoop11 bool) *Filter {
```

Reply:
Thanks, have reverted to cascading